website/public/js/controllers/sortableInventoryCtrl.js

```diff
@@ -0,0 +1,19 @@
+habitrpg.controller('SortableInventoryController', ['$scope',
+  function ($scope) {
+
+    // doing this in a function causes 10 $digest iteration error
```

Comment:
I don't think we need this comment. Declaring this object outside of the function has better performance implications by nature.